extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -276,29 +284,50 @@ export class InstrumentationService {
   }
 
   /**
+   * @param {!Element} el
+   * @private
+   * @return {Object}
+   */
+  extractAnalyticsVariables_(el) {
+    const dataSet = el.dataset;
+    const analyticsVariables = {};
+    for (const dataKey in dataSet) {
+      if (dataKey.indexOf(vars) === 0) {
+        const variableName = dataKey.toLowerCase().replace(vars,'');
```

Comment:
Update this to follow the same format of naming convention that is followed for other variables (camelCase) as in `domain=${canonicalHost}&path=${canonicalPath}`

`data-vars-sub-title="Example sub title"` will resolve to 
```
{
"varsSubTitle": "Example sub title"
} 
```
Which in turn is resolved to
```
{
"subTitle": "Example sub title"
} 
```